base/src/main/java/io/spine/validate/Validate.java

```diff
@@ -328,10 +93,13 @@ public static void checkValid(Message message) throws ValidationException {
      * @param context
      *         the validation field context
      * @return violations of the validation rules or an empty list if the message is valid
+     * @apiNote This method is used by the generated code, and as such needs to
+     *         be {@code public}.
      */
     @Internal
-    public static List<ConstraintViolation> validateAtRuntime(Message message,
-                                                              FieldContext context) {
+    @SuppressWarnings("WeakerAccess") // see apiNote.
+    public static
+    List<ConstraintViolation> validateAtRuntime(Message message, FieldContext context) {
```

Comment:
Maybe like this? I believe we use such a formatting a lot.
```suggestion
    public static List<ConstraintViolation> 
    validateAtRuntime(Message message, FieldContext context) {
```